dash_labs/plugins/pages.py

```diff
@@ -224,16 +236,17 @@ def _filename_to_name(filename):
     return filename.split(".")[-1].replace("_", " ").capitalize()
 
 
-def _filename_to_path(filename):
-    return filename.replace("_", "-").replace(".", "/").lower().split("pages")[-1]
-
+def _infer_path(filename, template):
+    if template is None:
+        path = filename.replace("_", "-").replace(".", "/").lower().split("pages")[-1]
+        path = "/" + path if not path.startswith("/") else path
+        return path
+    else:
+        # replace the variables in the template with "None"
+        return re.sub("<(.+?)>", "none", template)
```

Comment:
Note this is an older version.  Currently, it does essentially the same thing but does not use regex. https://github.com/plotly/dash-labs/blob/main/dash_labs/plugins/pages.py#L276

But no, it wont' matter if "none" is supplied anywhere else in the URL.  The only purpose of this is to infer a starting path if a `path_template` was supplied but not a `path`.  It creates a valid path by replacing the path variable names with "none".   I expect that the developer will then either handle this situation in the layout function, or supply a different path to `dash.register_page`.


If "none" is supplied elsewhere in the URL, it will go to a 404 page, unless it it happens to match some other valid path.